### PR TITLE
Ch setup docker and dockercompose

### PR DESCRIPTION
**Pull request by tuyishimejeandamour**

### CODE REVIEW

**Review by veritem — CHANGES_REQUESTED** [review]
This looks react good but we can we have Docker file at the root of the project?

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: ./docker/dev/Dockerfile
```
Comment (veritem) [inline docker-compose.yml:7]:
```suggestion
      dockerfile: ./Dockerfile
```
Having something like this and then have docker file at the root

File: docker/dev/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Base image
+FROM node:18-alpine
+
+# Set the working directory
+WORKDIR /app
+
+# Copy package.json and package-lock.json
+COPY ../../package.json .
```
Comment (veritem) [inline docker/dev/Dockerfile:8]: this would change to look like 
```suggestion
COPY ./package.json .
```

**Review by veritem — CHANGES_REQUESTED** [review]

File: docker/dev/Dockerfile (hunk 2 above)
Comment (tuyishimejeandamour) [reply docker/dev/Dockerfile:8]: that's right but according to project folder structure that's how it must be followed

Comment (veritem) [reply docker/dev/Dockerfile:8]: okay

**Review by veritem — APPROVED** [review]
if this is the project's convention, I'm going to merge this as approved

**Pull request merged**